argo_rollouts/datadog_checks/argo_rollouts/__about__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+__version__ = '1.0.0'
```

Comment:
```suggestion
__version__ = '0.0.1'
```

Sorry, we updated the template after you created the integration I believe